src/Http/Http.Results/test/EmptyResultTest.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Http.Result;
+
+public class EmptyResultTest
+{
+    public async Task EmptyResult_DoesNothing()
+    {
+        var emptyResult = EmptyResult.Instance;
+
+        // Assert- does not throw.
+        await emptyResult.ExecuteAsync(GetHttpContext());
```

Comment:
Can we assert a 200 status and that the Body MemoryStream hasn't been written to?

Reply:
Compared status code. Not sure if I compare streams correctly!?

Reply:
Looks good.